client-src/elements/chromedash-form-field.ts

```diff
@@ -504,5 +513,3 @@ function getFieldValueWithStage(fieldName, stageOrId, formFieldValues) {
   const value = getFieldValueFromFeature(fieldName, feStage, feature);
   return value;
 }
```

Comment:
Look for functions with `@param {...}` jsdoc, and convert them to Typescript syntax. Otherwise you actually lose typechecking during the conversion to TS.